src/encoder/ffmpeg/codec.hpp

```diff
@@ -37,6 +38,8 @@ class Codec {
 
   ContextPtr      m_context;
```

Comment:
`ContextPtr` should have been placed into `ffmpeg` namespace. (not issue of this PR, ofc, but it would be nice if...)